cardano-cli/src/Cardano/CLI/Shelley/Run/StakeAddress.hs

```diff
@@ -216,3 +172,45 @@ runStakeCredentialDeRegistrationCert stakeVerifier (OutputFile oFp) =
 
     deregCertDesc :: TextEnvelopeDescr
     deregCertDesc = "Stake Address Deregistration Certificate"
+
+
+loadStakeCredentialFromVerifier
+  :: StakeVerifier -> ExceptT ShelleyStakeAddressCmdError IO StakeCredential
+
+loadStakeAddressFromVerifier
+  :: NetworkId
+  -> StakeVerifier
+  -> ExceptT ShelleyStakeAddressCmdError IO StakeAddress
+
+(loadStakeCredentialFromVerifier, loadStakeAddressFromVerifier) =
```

Comment:
This is quite a confusing way to define these functions. Let's separate them. 

I am curious though, why did you define them like this?

Reply:
They have common internal logic that should be hidden.

Alternative: define 3 functions, but the 3rd one will be visible to all other functions in the module.

Reply:
Yes please define three separate functions.